### DISCUSSION TITLE
vertical lxqt-panel is overlapping desktop icons

### DISCUSSION THREAD

**Discussion by Lokawn** [lxqt/lxqt#2255] [Q&A] [answered]
![screengrab-2022-06-18-10-33-08](https://user-images.githubusercontent.com/39772448/174424015-1a625ebf-45ef-4f67-8bd7-27e67a46d41c.png)

Desktop Icons are being overlapped by the vertical lxqt-panel. How can I make it so that the desktop icons shift to give the panel some space? the panel  auto hides but it doesn't on the desktop (which is the correct behaviour), so the icons are overlapped.
I can make the icons sticky but then it looks a little awkward.

The Home directory icons should shows username instead of HOME or the user's name.

**Comment by stefonarch** [comment]
Setting to max value the horizontal spacing is atm afaik the only way.

![screenshot-558787e4](https://user-images.githubusercontent.com/10681413/174426131-a5d306f4-d861-4d7e-86e6-f0782464616a.png)

**Reply by Lokawn** [reply]
Thanks.
How can I rename the home directory on desktop it just shows open, do I need to create a new launcher Item instead of enabling it in Desktop Preferences?

**Reply by stefonarch** [reply]
The home dir on the desktop has the username by default... Check in "session settings" the directories.

**Comment by tsujan** [comment, marked as answer]
> Setting to max value the horizontal spacing is atm afaik the only way.

Using large item margins has this drawback that there will be large vertical or horizontal spaces between *all* items.

I'll see if I could add a new option to pcmanfm-qt's Desktop Preferences for *work area margin(s)*.

This is a valid feature request. I wonder why it wasn't reported before!

**Reply by stefonarch** [reply]
One solution could be that those values would be left and top distances. 
But yes, left and top panels could need such an option. Top panels usually are tiny so this issue doesn't show up.

**Reply by tsujan** [reply]
Yes, it'll be an option for 4 margins of the work area: left, top, right and bottom. The code already reserves 12px for all. So, adding an option should be very easy.

I have a thin top panel that reserves space and a thick bottom panel that hides on overlapping windows. The bottom panel, which I use like a dock, doesn't cover the whole screen width but, sometimes, desktop items go behind it. That's like what @Lokawn reported here. That above-mentioned option will solve such problems.

**Reply by tsujan** [reply]
Something like this:

![margins](https://user-images.githubusercontent.com/981076/174446599-545a7168-0cd6-49e1-ad95-a6fedfaa6a68.png)

**Reply by stefonarch** [reply]
That looks good!
"work area" → "Desktop"?

**Reply by tsujan** [reply]
> "work area" → "Desktop"?

"Work area" is a part of Desktop that is actually used. Desktop ≥ Work Area

**Reply by tsujan** [reply]
Done in https://github.com/lxqt/pcmanfm-qt/pull/1606

**Reply by Lokawn** [reply]
Thanks, super fast.
